torch/nn/modules/upsampling.py

```diff
@@ -109,6 +111,104 @@ class UpsamplingBilinear2d(_UpsamplingBase):
         [torch.FloatTensor of size 1x1x4x4]
 
     """
+    def __init__(self, size=None, scale_factor=None):
+        super(UpsamplingBilinear2d, self).__init__(_pair(size), scale_factor)
```

Comment:
pass in size instead of _pair(size), and cast self.size() after the parent constructor